code/modules/mob/living/simple_animal/hostile/spider_queen.dm

```diff
@@ -127,4 +131,5 @@
 
 /mob/living/simple_animal/hostile/giant_spider/nurse/spider_queen/Life(seconds_per_tick, times_fired)
 	..()
-	adjustBruteLoss(-3)
+	if(stat != DEAD)
```

Comment:
Make this a guard statement like in `/mob/living/simple_animal/Life(seconds_per_tick, times_fired)`

Reply:
Implemented.